packages/kit/src/core/adapt/builder.js

```diff
@@ -190,7 +193,10 @@ export function create_builder({
 		},
 
 		writeServer(dest) {
-			return copy(`${config.kit.outDir}/output/server`, dest);
+			const files = copy(`${config.kit.outDir}/output/server`, dest);
+			// remove unused vite files
```

Comment:
I'm not sure I'd remove it on the server actually. It'd be kind of helpful to leave it for debugging

Reply:
removed in the latest commit